src/github.ts

```diff
@@ -66,6 +66,21 @@ export const fetchTargeting = async (branch: string) => {
   return json;
 };
 
+// returns a list of closed PRs that have the given milestone
+export const fetchUnmergedClosedWithMilestone = async (
+  milestoneTitle: string,
+) => {
+  const response = await fetch(
+    `${GITHUB_API}/search/issues?q=` +
+      encodeURIComponent(
+        `is:pr is:closed is:unmerged milestone:${milestoneTitle} repo:go-gitea/gitea`,
+      ),
+    { headers: HEADERS },
+  );
+  const json = await response.json();
+  return json;
```

Comment:
Does this API return `[]` when no matches?

Reply:
Yes

Reply:
Ah, what luxure when one has a well-behaved API to talk to 😆

Reply:
(For some reason, I can not resolve comments in this repo)

Reply:
GitHub's API is pretty good IMO